packages/react/index.stable.js

```diff
@@ -23,6 +23,8 @@ export {
   useDebugValue,
   useLayoutEffect,
   useMemo,
+  useMutableSource,
+  createMutableSource,
```

Comment:
Ah, almost missed this. Let's remove these from stable until we're ready to release them.

Reply:
Ah, right.